get_github_coderabbit_review/tests/integration/test_github_integration.py

```
@@ -0,0 +1,465 @@
+            )
+
+        mock_run.side_effect = slow_response
+
+        results = []
+        errors = []
+
+        def fetch_comments():
+            try:
+                result = self.client.fetch_pr_comments("https://github.com/owner/repo/pull/123")
+                results.append(result)
+            except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)